schema.graphql

```diff
@@ -4068,6 +4068,7 @@ type Invoice {
 type InvoiceAppliedTax implements AppliedTax {
   amountCents: BigInt!
   amountCurrency: CurrencyEnum!
+  appliedOnWholeInvoice?: Boolean!
```

Comment:
is the schema up-to-date? I can't see `taxCodes` for instance 🤔

Reply:
hm
that's interesting as I added both fields and run the command to regenerate the schema...
will fix it shortly

Reply:
thanks for noticing!

Reply:
found it, thanks!
the InvoiceAppliedTax implements AppliedTax, which has taxCode, so we don't need to add it on the InvoiceAppliedTax level, cleaned 😁